### DISCUSSION TITLE
Select input createOptionForm outside panel

### DISCUSSION THREAD

**Discussion by gsusanj** [filamentphp/filament#9440] [Help] [answered]
Hi everyone,
I am not sure if its an isolated issue or expected behaviour, I am having trouble of using the createOptionForm on an input element.
The + button appears, but when I click it, it loads for a while and then nothing happens.

Note, I am trying to use this option outside of the panel, maybe I need to create a custom modal container where this form shows up? If yes, how do I do that? Through createOptionAction?

```
  Select::make( 'field' )
                ->options( Model::pluck('name', 'id') )
                ->searchable()
                ->live()
                ->createOptionForm( [
                                TextInput::make('name),
                ] )
                ->createOptionUsing( fn( $data ) => dd( $data ) )
                ->createOptionAction( fn( Action $action ) => $action),
```

**Comment by leandrocfe** [comment, marked as answer]
Did you add this?
```html
<x-filament-actions::modals />
```

**Reply by gsusanj** [reply]
A life saver, thank you. I have searched the documentation thoroughly but without success. Now searching this I did find where it is.

**Reply by maccarib** [reply]
I have the same challenge, but cannot really work out where exactly I would have to add this and if there's anything else that needs to be done? Can you please provide me with a hint?
